DWIConvert/TestSuite/CMakeLists.txt

```diff
@@ -130,6 +136,10 @@ set(TstOutput ${CMAKE_CURRENT_BINARY_DIR})
 
 set(DWIBASELINE_DIR ${TestData_DIR}/DWI_TestData_OUTPUTS)
 
+## DUMMYPROG
```

Comment:
This was a one time use script.  It should not be compiled every night, or perhaps ever again.

If you wish to keep the code for historical purposes, it needs to be commented-out, or moved somewhere else.

This is probably not worth keeping around in it's current form.  Copy and paste it into one of your OneNote notebooks and keep it out of BRAINSTools